### DISCUSSION TITLE
Not receiving notifications

### DISCUSSION THREAD

**Discussion by Goddess0098** [community/community#143878] [Mobile]
`BF2A:1C353F:C3CF40:C6DB
D3:672B2872'![Screenshot_2024-11-06-11-57-44-677_com.github.android.jpg](https://github.com/user-attachments/assets/193e1d6e-9189-4248-8f23-0f3f08ec55ec)

**Comment by NabilMch** [comment]
Try deleting and reinstalling the app; it should work afterward. Don’t forget to grant the necessary permissions to the app on your phone!

**Comment by Goddess0098** [comment]
> `BF2A:1C353F:C3CF40:C6DB
> D3:672B2872'![Screenshot_2024-11-06-11-57-44-677_com.github.android.jpg](https://github.com/user-attachments/assets/193e1d6e-9189-4248-8f23-0f3f08ec55ec)
> 
> 

Thank you very much, my problem was solved
